src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -123,13 +122,6 @@ public final class EnhancedQueueExecutor extends AbstractExecutorService impleme
     // Optimization control flags
     // =======================================================
 
-    /**
-     * A global hint which establishes whether it is recommended to disable uses of {@code EnhancedQueueExecutor}.
-     * This hint defaults to {@code false} but can be changed to {@code true} by setting the {@code jboss.threads.eqe.disable}
-     * property to {@code true} before this class is initialized.
-     */
-    public static final boolean DISABLE_HINT = readBooleanPropertyPrefixed("disable", false);
-
```

Comment:
This is part of the API, so removing it would be a problem. We could deprecate it and set it to `false` instead though.

Reply:
let me remove from this PR than, better track it separately